test/powershell/Modules/Microsoft.PowerShell.Utility/Send-MailMessage.Tests.ps1

```diff
@@ -0,0 +1,147 @@
+Describe "Basic Send-MailMessage tests" -Tags CI {
```

Comment:
Does this test require Administrator on Windows or root on all non-Windows platforms?